packages/adapter-vercel/image-loader.js

```diff
@@ -0,0 +1,49 @@
+// https://vercel.com/docs/concepts/image-optimization
+
+/**
+ * @param {string} src
+ * @param {number} width
+ * @param {{ quality?: number }} [options]
+ */
+export default function loader(src, width, options) {
+	const url = new URL(src, 'http://n'); // If the base is a relative URL, we need to add a dummy host to the URL
+	if (url.pathname === '/_vercel/image') {
+		set_param(url, 'w', width);
+		set_param(url, 'q', options?.quality ?? 75, false);
```

Comment:
this is rather surprising to me. I'd expect if the `quality` option is not set then the `q` parameter should not be set either rather than defaulting to `75`